app/src/test/java/com/oracle/javafx/scenebuilder/app/JfxInitializer.java

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Too bad we need to duplicate this class in `app`. Publishing test-kit for the sake of adding it as a dependency might be overkilling just for this?

Reply:
Actually, as of now, I'd like to avoid adding test-kit as a dependency. If we get more tests in app, its probably worth to think about what we could reuse from test-kit.